packages/flutter_tools/lib/src/android/android_sdk.dart

```diff
@@ -202,9 +205,42 @@ class AndroidNdk {
       return <String>['--sysroot', armPlatform];
     }
 
+    int findNdkMajorVersion(String ndkDirectory) {
+      final String propertiesFile = fs.path.join(ndkDirectory, 'source.properties');
+      if (!fs.isFileSync(propertiesFile)) {
+        throw AndroidNdkSearchError('Can not establish ndk-bundle version: ${propertiesFile} not found');
+      }
+
+      // Parse source.properties: each line has Key = Value format.
+      final Map<String, String> properties = Map<String, String>.fromIterable(
```

Comment:
Add a comment how the string looks like you're looking for

    // Pkg.Revision = <ndk-version>.1.5063045